libs/partners/databricks/.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
Isn't this ignored in the root `.gitignore`?

Reply:
It's just auto-generated and other packages seems to have the same file. I'd just keep it untouched:)